BlueEastCode/bluerain-debugger-sentry
# feat(README.md): Update the README.md

## What I did
Update the README.md
## How to test
npm run test
Is this testable with jest or storyshots?
yes
Are any of the existing tests are failing ?
no
Does this need an update to the documentation?
I update the documentation already not now.
Does it have breaking changes ?
No
If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
